GAL22v10_counter8.pld

```diff
@@ -0,0 +1,95 @@
+GAL22V10
+Counter
+
+Clock NC    NC    NC    NC    NC    NC    NC    NC    NC     Reset     GND
+/OE   Q0    Q1    Q2    Q3    Q4    Q5    Q6    Q7    RO     NC     VCC
+
+AR = Reset
+
+Q0.R = /Q0
+
+; 0 1
+; 1 0
+Q1.R = /Q1 * Q0 + Q1 * /Q0
+
+; 0 1 1
+; 1 0 x
+; 1 x 0
+Q2.R = /Q2*Q1*Q0 +
+    Q2 * /Q1 +
+    Q2 * /Q0
```

Comment:
Love this! In order to check my understanding, I want to build a logical equivalence to what I did in the 6-bit horizontal counter. Here's my attempt: in order for the later bits to "matter," the higher bits have to be 1. So, the `Σ Q_N * Q_N-1 .. * /Q_M` strategy ends up with at most one 1 going into the sum term, whereas this will OR together a lot of ones when e.g. Q = 0b100. But 0+1+1 is the same result as 0+1+0 in the boolean ring (field?), so they're equivalent functions.

I like this much better!